ONSdigital/eq-author-app
# Section Introduction Review

### What is the context of this PR?
Change to move section introduction to be fields in the section and to hide the section title when the section navigation is disabled.

### How to review 
1. Ensure that the section title is hidden unless section navigation is enabled
2. Ensure that the section intro can be updated and still seen in publisher.
